css-canvas-medical/styles.css

```diff
@@ -0,0 +1,291 @@
+h2 {
+  position: absolute;
+  top: 5rem;
+  left: 4rem;
+  font-size: 32px;
+  color: white;
+  font-family: 'Tiempos headline', sans-serif;
+  font-weight: 300;
+}
+
+h3 {
+  margin-bottom: 0;
+  font-family: 'Tiempos Headline', sans-serif;
+}
+
+h6 {
+  text-transform: uppercase;
+  color: white;
+  font-family: 'Avenir', sans-serif;
+  margin-bottom: 5px;
+}
+
+.container {
+  width: 66%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.footer-color {
+  background-color: #fafafa;
+}
+
+.link {
+  text-decoration: none;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.justify-between {
+  justify-content: space-between;
+}
+
+.justify-around {
+  justify-content: space-around;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.flex-row {
+  flex-direction: row;
+}
+
+.col-fifth {
+  width: 20%;
+}
+
+.col-full {
+  width: 28%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.col-center {
+  display: flex;
+  align-items: center;
+}
+
+.col-4ths {
+  display: flex;
+  flex-direction: column;
+  width: 25%;
+  justify-content: center;
+}
+
+.col-5 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 50%;
+  margin-left: 3rem;
+}
+
+.col-half {
+  width: 50%;
+}
+
+.address {
+  padding: 30px;
+  margin: auto;
+  font-size: 11px;
+  align-items: center;
+}
+
+.row-1 {
+  width: 100%;
+  margin: auto;
+  justify-content: space-between;
+  margin-top: 10px;
+  border-bottom: 1px solid #ebecef;
+  padding-bottom: 15px;
+}
+
+.row-3 {
+  background-color: #fafafa;
+}
+
+.row-4 {
+  width: 90%;
+  margin: 10px auto;
+}
+
+.row-5 {
+  background-color: #525351;
+  height: 50vh;
+}
+
+.row-6 {
+  width: 100%;
+  margin: auto;
+  padding: 40px;
+}
```

Comment:
I think these classes would be better named after their content, since they are specific to specific sections. Good classnames would be things like `header-row`, `hero-img-row`, etc.